src/core/src/enclave_sgx.rs

```diff
@@ -166,13 +167,14 @@ impl EnclaveConfig {
         self.signup_info.enclave_quote = signup.enclave_quote;
 
         let (poet_public_key, quote) = self.get_signup_parameters();
+        let nonce = &sha512_from_str(poet_public_key.as_str())[..MAXIMUM_NONCE_LENGTH];
         let mut proof_data_string = String::new();
-        let mut epid_pseudonym = String::new();
+        let mut epid_pseudonym = poet_public_key.clone(); // String::new();
```

Comment:
This is temporary change, until we have mock client and mock AVR for VR TP. VR TP makes use of this field to store validator_map entry.